oracle-plugin/docs/Oracle-batchsink.md

```diff
@@ -30,6 +30,8 @@ You also can use the macro function ${conn(connection-name)}.
 
 **SID/Service Name:** Oracle connection point (Database name or Service name).
 
+**Role** Login role of the user when connecting to the database. For eg, NORMAL, SYSDBA, SYSOPER, etc.
```

Comment:
@itsankit-google If you want to be consistent with the Oracle connection and Oracle batch source markdown they say "Login role of the user when connecting to the database."